### PR TITLE
Upgrade euneus to 2.3 - add OTP27 to tests

### PR DESCRIPTION
**Pull request by mworrell**
TODO:

 - [x] Fix error handler (skipping unknown values)

### CODE REVIEW

**Comment by mworrell** [comment]
@williamthome do you know of a method to not include the json-polyfill for OTP27+ ?

File: rebar.config [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {require_min_otp_vsn, "21"}.
 
 {deps, [
-    {euneus, "0.6.0"}
+    {euneus, "~>2.3"},
+    {json_polyfill, "0.1.3"}
```
Comment (williamthome) [inline rebar.config]:
```suggestion
    {json_polyfill, "0.1.4"}
```

Please use [v0.1.4](https://github.com/williamthome/json_polyfill/releases/tag/v0.1.4), and you are good to go with OTP-27.

**Review by williamthome — CHANGES_REQUESTED** [review]

File: rebar.config (hunk 1 above)
Comment (mworrell) [reply rebar.config]: Great!  Giving it a go :-)

Comment (mworrell) [reply rebar.config]: Looking at your change, a pity that such a hack is needed for a OTP version-dependency.

Comment (williamthome) [reply rebar.config]: Yes :sweat_smile: 

Having a min and max OTP version option for deps for such a case would be nice.

**Review by williamthome — APPROVED** [review]
LGTM o/

**Pull request merged**